packages/app/src/pages/dashboard/history.tsx

```diff
@@ -1,27 +1,51 @@
 import Head from 'next/head'
-import { ReactNode } from 'react'
+import { ReactNode, useCallback, useEffect, useState } from 'react'
 import { useTranslation } from 'react-i18next'
 
 import { MobileHiddenView, MobileOnlyView } from 'components/Media'
 import Spacer from 'components/Spacer'
 import DashboardLayout from 'sections/dashboard/DashboardLayout'
+import HistoryTabs, { HistoryTab } from 'sections/dashboard/HistoryTabs'
 import TradesTab from 'sections/futures/MobileTrade/UserTabs/TradesTab'
-import Trades from 'sections/futures/Trades'
 import { usePollDashboardFuturesData } from 'state/futures/hooks'
+import { setSelectedTrader } from 'state/futures/reducer'
+import { useAppDispatch, useAppSelector } from 'state/hooks'
+import { selectWallet } from 'state/wallet/selectors'
 
 type HistoryPageProps = React.FC & { getLayout: (page: ReactNode) => JSX.Element }
 
 const HistoryPage: HistoryPageProps = () => {
 	const { t } = useTranslation()
+	const dispatch = useAppDispatch()
+	const walletAddress = useAppSelector(selectWallet)
+
 	usePollDashboardFuturesData()
+
+	const [currentTab, setCurrentTab] = useState(HistoryTab.Positions)
+
+	const handleChangeTab = useCallback(
+		(tab: HistoryTab) => () => {
+			setCurrentTab(tab)
+		},
+		[]
+	)
+
+	useEffect(() => {
+		dispatch(
+			setSelectedTrader({
```

Comment:
I think we should reserve the use of this for leaderboard rather than mix it into user history.